includes/class-wporg-notifications.php

```diff
@@ -325,6 +325,19 @@ public static function get_email_body( WP_Comment $comment, ?array $comment_meta
 				$output .= '- <strong>' . esc_html__( 'Translation string: ' ) . '</strong>' . esc_html( $translation->translation_0 ) . '<br>';
 			}
 		}
+		if ( ! empty( $comment_meta['reject_reason'][0] ) ) {
```

Comment:
```suggestion
		if ( ! empty( maybe_unserialize( $comment_meta['reject_reason'][0] ) ) ) {
```

We need the same change in the DotOrg environment.

![image](https://user-images.githubusercontent.com/1667814/187562213-b0cb6f1b-e742-4065-aebd-e524e36d1937.png)

Reply:
Same as above
```suggestion
		if ( isset( $comment_meta['reject_reason'][0] ) && ! empty( maybe_unserialize( $comment_meta['reject_reason'][0] ) ) ) {
```